opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -422,6 +446,24 @@ def get_laser_envelope( self, t=None, iteration=None, pol=None, m='all',
         if index != 'all':
             info.restrict_to_1Daxis( info.axes[1] )
 
+        # Plot the result if needed
+        if plot:
+            iteration = self.iterations[ self.current_i ]
+            time_fs = 1.e15 * self.t[ self.current_i ]
+            if index != 'all':
+                plt.plot( info.z, envelope, **kw)
+                plt.ylabel('$E_%s \;(V/m)$' % pol,
+                           fontsize=self.plotter.fontsize)
+            else:
+                plt.imshow( envelope, extent=info.imshow_extent,
```

Comment:
Similarly, `info.imshow_extent` is also in SI. Please use `1.e6*info.imshow_extent`.

Btw, Sorry for these inconsistencies in openPMD-viewer: some quantities are in microns and some other are in SI. My aim is to uniformize all of this in the future.